README.md

```diff
@@ -13,36 +13,76 @@ for use within the [HCP Minimal Preprocessing Pipelines][HCP Pipelines].
 
 ## Installation
 
-### Prerequisites
-
+### Install for all users
 You can install the necessary prerequisites in most Ubuntu or Debian-based distributions with this command:
-```
-sudo apt-get install python3-numpy python3-pip
+```bash
+sudo apt install python3-numpy python3-pip
 ```
 
-### Install
-
 For convenience the latest gradunwarp tarball can be downloaded from [here][gradunwarp-hcp-tarball].
 
 First, extract the gradunwarp tarball, and `cd` into the folder it creates. Then do:
-```
+```bash
+sudo pip3 install -r requirements.txt
 sudo pip3 install .
 ```
-If you don't have superuser permissions on the machine, you can use the `--user` switch of pip install:
-```
+
+#### Install with only user permissions
+If you don't have superuser permissions on the machine, you can use the `--user` switch of pip install instead of using `sudo`:
+```bash
+pip3 install -r requirements.txt --user
 pip3 install . --user
 ```
 If you use the `--user` switch, you will need to add `/home/<username>/.local/bin` to your `PATH` environment variable, replacing `<username>` with your user name.
 
-#### Details
+### Install using a virtual environment
+If the [python virtual environment module](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments) is not already installed, then you may need to run
```

Comment:
This might be confusing to users, as it may suggest that a virtualenv is required.  Can we add a sentence like "You may optionally choose to install gradunwarp into a python virtual environment in order to avoid changing your existing python setup"?